jobs/load-common-data-from-openmrs.yml

```diff
@@ -45,3 +47,4 @@ configuration:
           - tableName: "users"
           - tableName: "user_roles"
           - tableName: "user_logins"
+            conditional: "select table_exists('authentication_event_log')"
```

Comment:
For this one iteration - for the user_logins table - this sets the conditional so that this job only runs if the authentication_event_log table exists (new function added to pihemr config in earlier commit)